SoundCloud.API.Client/Internal/Infrastructure/Serialization/JsonSerializer.cs

```diff
@@ -14,7 +14,14 @@ public string Serialize<T>(T obj)
 
         public T Deserialize<T>(string json)
         {
-            return JsonConvert.DeserializeObject<T>(json, customConverters);
+            try
```

Comment:
Oh, sorry, this isn't needed anymore. It's left in from debugging when I was trying to figure out why some Tracks were failing to be deserialized.

Turns out that SoundCloud returns Track objects with some null fields sometimes, and Newtonsoft.JSON didn't handle these by default. That's why I added all the `NullValueHandling = NullValueHandling.Ignore` attributes